HISTORY.md

```diff
@@ -1,3 +1,11 @@
+### 1.23.1
+
+* Bump standard from 16.0.4 to 17.0.0
```

Comment:
Don't need to include things that aren't relevant to the user, just changes that are relevant to API users

Reply:
so then remove everything except the 1.19 change, or is there something else relevant?

Reply:
fixed